src/js/angular/core/directives/rdf-resource-search.directive.js

```diff
@@ -0,0 +1,66 @@
+angular
```

Comment:
Not a single test for this directive? We should at least check if it's present on each page and its behavior: open, close, search, etc.